src/include/migraphx/match/gelu_erf.hpp

```diff
@@ -67,6 +67,41 @@ inline auto gelu_erf()
     return gelu_erf([](auto x) { return name(x); });
 }
 
+namespace detail {
+template <class F>
+struct bert_gelu_erf_matcher
```

Comment:
Are you planning on renaming this to `gelu_erf_matcher`?

Reply:
I added this one because I had to make a small change to get it to match the way that gelu appears in huggingface bert. Is there another way you'd prefer to differentiate the two?